solutions/devsprint-luiz-zytkowski-2/src/main/kotlin/io/devpass/parky/service/ParkingSpotService.kt

```diff
@@ -1,9 +1,14 @@
 package io.devpass.parky.service
 
+import io.devpass.parky.entity.ParkingSpot
 import io.devpass.parky.repository.ParkingSpotRepository
 import org.springframework.stereotype.Service
 
 @Service
 class ParkingSpotService(
-    private val parkingSpotRepository: ParkingSpotRepository,
-)
\ No newline at end of file
+    private val parkingSpotRepository: ParkingSpotRepository
+) {
+    fun findAll() : MutableIterable<ParkingSpot> {
```

Comment:
Para fins de simplicidade, vamos determinar o retorno como ```List<ParkingSpot>``` 😉 está certo o seu código, só que esse ```MutableIterable<ParkingSpot>``` pode confundir quem for consumir o método!